dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/condition/HttpUrlCondition.kt

```diff
@@ -0,0 +1,23 @@
+package pl.droidsonroids.testing.mockwebserver.condition
+
+import okhttp3.HttpUrl
+import okhttp3.mockwebserver.RecordedRequest
+
+/**
+ * A Condition which matches HttpUrls only, they are extracted from request so that only
+ * <code>isUrlMatching(HttpUrl)</code> needs to implemented.
```

Comment:
to jest kdoc, nie javadoc, tu nie ma @code ;)
https://kotlinlang.org/docs/reference/kotlin-doc.html
ale można użyć `[]` sprawdzę tę opcję